### ISSUE TITLE
update database table structure failed on v0.4.0

### ISSUE THREAD

**Issue by WLun001** [mayswind/ezbookkeeping#21]
```
2023-09-18 01:04:43 [INFO] [server.startWebServer] static root path is /ezbookkeeping/public
2023/09/18 01:04:43 Failed to run ezBookkeeping with [/ezbookkeeping/ezbookkeeping server run]: Cannot add a NOT NULL column with default value NULL
2023-09-18 01:04:43 [ERROR] [server.startWebServer] update database table structure failed, because Cannot add a NOT NULL column with default value NULL
2023-09-18 01:05:10 [INFO] [initializer.initializeSystem] will load configuration from default config path /ezbookkeeping/conf/ezbookkeeping.ini
```

**Comment by mayswind** [comment]
fixed in v0.4.1

**Comment by WLun001** [comment]
tested, working fine, thanks!
